### DISCUSSION TITLE
Error: PocketBase instance failed to launch.

### DISCUSSION THREAD

**Discussion by moussa32** [pockethost/pockethost#374] [Q&A]
Hello, I was building an app on localhost using pocketbase v0.22.2 I got a backup of data & collections then imported collections first in my pockethost.io instance and it did it correctly without problems but when I uploaded my records backup. the dashboard failed and got the error message `Error: PocketBase instance failed to launch. Please check logs at https://app.pockethost.io. [Sensitive Credential]. https:/pockethost.io/docs/usage/errors`
this is the logs I got in pockethost dashboard

```
2024-03-07 Starting instance
2024-03-07 2024/03/07 09:02:52 ***defaults successfully applied
2024-03-07 Error: Failed to apply migration 1709801984_created_platforms.js: constraint failed: UNIQUE constraint failed: _collections.name (2067)
2024-03-07 Process exited with code 0
2024-03-07 Error spawning: Error: Container stopped
```

And I am stuck I can't open the dashboard or delete the backup or anything just a blank page with the error message

**Comment by benallfree** [comment]
Hello, I moved your issue to the discussion area. Please note that most of us are on Discord :) 

It looks like one of your migrations is failing (`1709801984_created_platforms.js`). You must go in via FTP and remove/disable it before your instance will launch again.
